templates/accounts/volunteer_profile.html

```diff
@@ -229,7 +229,18 @@ <h2 class="font-weight-bolder">Complete Your Profile</h2>
         }, false)
     }())
 </script>
-
+<script type="text/javascript">
+    function ImageValidation(){
+    var fileInput = document.getElementById('profileImage');
+    var filePath = fileInput.value;
+    var allowedExtensions = /(\.jpg|\.jpeg|\.png|\.gif)$/i;
+    if(!allowedExtensions.exec(filePath)){
+        alert('Please upload file having extensions .jpeg, .jpg, .png or .gif only.');
+        fileInput.value = '';
+        return false;
```

Comment:
No need to return anything here. This is important only when it is used in `onsubmit` event (to stop the form from submitting).